configuration/configuration.py

```diff
@@ -120,6 +120,11 @@ def _read_secret(secret_name, default = None):
 CORS_ORIGIN_WHITELIST = list(filter(None, environ.get('CORS_ORIGIN_WHITELIST', 'https://localhost').split(' ')))
 CORS_ORIGIN_REGEX_WHITELIST = [re.compile(r) for r in list(filter(None, environ.get('CORS_ORIGIN_REGEX_WHITELIST', '').split(' ')))]
 
+# Cross-Site-Request-Forgery-Attack settings. If Netbox is sitting behind a reverse proxy, you might need to set the CSRF_TRUSTED_ORIGINS flag.
+# Django 4.0 requires to specify the URL Scheme in this setting. An example environment variable could be specified like:
+# CSRF_TRUSTED_ORIGINS=https://demo.netbox.dev http://demo.netbox.dev
+CSRF_TRUSTED_ORIGINS = list(filter(None, environ.get('CSRF_TRUSTED_ORIGINS', 'https://localhost').split(' ')))
```

Comment:
The default-value of upstream is empty. What is the reason that you've added `https://localhost` here as the default?

https://github.com/netbox-community/netbox/blob/ea197eff5f4fe925bb354d1375912decd81752bd/netbox/netbox/settings.py#L80

Reply:
Sorry, my bad - copy and pasted it from the line above. I've fixed it.